src/components/Avatar/Avatar.test.js

```diff
@@ -0,0 +1,10 @@
+import Avatar from '.';
+import { screen } from '@testing-library/react';
+
+describe('Avatar', () => {
+    test('it shows a users avatar', () => {
+        renderWithReduxProvider(<Avatar />)
+        const image = screen.getByRole('img', { alt: "github-user-avatar" })
+        expect(image).toBeInTheDocument
+    })
+})
```

Comment:
I love that you've implemented some testing too!  (On an aside, you could maybe mention in the README that there is a working test suite!)